Dockerfile

```diff
@@ -8,12 +8,27 @@ RUN rm -r /context/etc
 
 
 FROM base_image AS base_image_python
+
 RUN apt-get update && \
     apt-get full-upgrade -y --no-install-recommends && \
     apt-get install -y --no-install-recommends python3-pip && \
     python3 -m pip install --no-cache-dir -U pip setuptools wheel
-#RUN apt-get install -y --no-install-recommends wget libcap2-bin
 
+RUN apt-get install -y --no-install-recommends wget libcap2-bin
+
+ENV PATH="/root/miniconda3/bin:${PATH}"
+ARG PATH="/root/miniconda3/bin:${PATH}"
+RUN wget \
+    https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+    && mkdir /root/.conda \
+    && bash Miniconda3-latest-Linux-x86_64.sh -b \
+    && rm -f Miniconda3-latest-Linux-x86_64.sh
+
+COPY resilience-env.yml .
+RUN . /root/.bashrc && \
```

Comment:
Does `. /root/.bashrc` actually do something? Nothing modifies it, as far as I can tell.

Reply:
In fact, it appears to cause some random errors:

```
/bin/sh: 13: /root/.bashrc: shopt: not found
/bin/sh: 21: /root/.bashrc: shopt: not found
```

Docker uses `sh`, but `.bashrc` is for `bash`.

Reply:
The activate doesn't work without it. Nevertheless to your point, `activate` in RUN has temporary effect. How do I activate the environment permanently? I guess I need to make it the conda default env and call conda in bashrc. Not sure. I don't like the fact that it complains about `conda init`. That should have a lasting side-effect on the file-system.

Reply:
Answering my own question: https://pythonspeed.com/articles/activate-conda-dockerfile/

Reply:
BTW `conda init bash` touches that file:
``` 
=> [base_image_python 5/5] RUN . /root/.bashrc &&     conda init bash &&     conda env create -n resilience --file resilience-e  295.7s
 => => # no change     /root/miniconda3/shell/condabin/conda-hook.ps1                                                                   
 => => # no change     /root/miniconda3/lib/python3.7/site-packages/xontrib/conda.xsh                                                   
 => => # no change     /root/miniconda3/etc/profile.d/conda.csh                                                                          
 => => # modified      /root/.bashrc
```